app/views/phases/_guidances_notes.html.erb

```diff
@@ -70,6 +70,7 @@
         <% guidance_accordion_id = 0 %>
         <% guidance_set.keys.each do |group| %>
```

Comment:
if we use each_with_index, we don't need to create an extra variable and keep the counter updated. For instance:
```
guidance_set.keys.each_with_index do |key, i| do
bla bla bla
end
```

Reply:
👍